app/assets/javascripts/discourse/initializers/android-app-banner-service-worker.js.es6

```diff
@@ -0,0 +1,22 @@
+// Android Chrome App Banner requires at least **one** service worker to be instantiate.
+// After Discourse starts to use service workers for other stuff (like mobile notification, offline mode, or ember)
+// we can ditch this.
+
+export default {
+  name: 'android-app-banner-service-worker',
+
+  initialize(container) {
+    const caps = container.lookup('capabilities:main');
+    if (caps.isAndroid) {
+      if ('serviceWorker' in navigator) {
+        navigator.serviceWorker.register(Discourse.BaseUri + '/service-worker.js', {scope: './'})
+          .then(function (r) {
+            // Service Worker Success
+          })
+          .catch(function (whut) {
```

Comment:
Actually if we're not doing anything with the response, we should leave out the calls. `then` and `catch`